catalog/core/catalog-core-backupplugin/pom.xml

```diff
@@ -27,6 +29,22 @@
             <groupId>ddf.catalog.core</groupId>
             <artifactId>catalog-core-api-impl</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.hamcrest</groupId>
+            <artifactId>hamcrest-all</artifactId>
+            <version>1.3</version>
```

Comment:
I  it , but it broke a bunch of stuff. And it did it in an ugly way. I might have to write a python or sed script to switch the order of arguments for a bunch of assertThat calls. So I punted on it and just updated the local POM. There is another ticket to update our versions of hamcrest, junit, and mockito. I'll tackle it then.